### ISSUE TITLE
Print page and save to PDF

### ISSUE THREAD

**Issue by marios88** [chrome-php/headless-chromium-php#49]
I read you have the issue in your TODO list

I am currently using the command line of chrome to generate PDFs but its very lacking in options compared to the puppeteer version. Since i dont have nodejs in my current env, i am stuck with it :(

My use case is, that i generate the HTML and feed it to chrome and grab the resulting file

Since this library supports sending direct commands to chrome,
has anyone successfully generated and saved PDFs?

Thank you for your time

**Comment by gsouf** [comment]
Hi @marios88 

That's not implemented yet. Might be done quickly though

**Comment by tanasecosminromeo** [comment]
@marios88 @gsouf Happy to make a PR, currently you can only do basic print as pdf settings (default, set orientation and paper size) but we could only improve later I think

**Comment by tanasecosminromeo** [comment]
@gsouf Please check https://github.com/chrome-php/headless-chromium-php/pull/50

**Comment by marios88** [comment]
@tanasecosminromeo 
Indeed it seems easy to implement the rest of the options. i will test it within the week and report back

**Comment by gsouf** [comment]
@marios88 that's available in version ``0.6.0``:

```php
$page->pdf([
    'landscape' => true,
    'printBackground' => true
])->saveToFile('/tmp/file.pdf');
```
